### DISCUSSION TITLE
Avoid patch on the conan recipe

### DISCUSSION THREAD

**Discussion by MartinDelille** [getsentry/sentry-native#808] [General]
As I was upgrading the conan recipe version to 0.6.0 I noticed that there is currently a patch for the sentry-crashpad recipe on CCI: https://github.com/conan-io/conan-center-index/blob/master/recipes/sentry-crashpad/all/conanfile.py#L99-L102

Could we avoid this patch in a future version ? I know that this is a forked repository but I ask just in case.

**Comment by supervacuus** [comment]
Hi @MartinDelille, I'm sorry for letting you wait. I have been sick for the last two weeks. I am not 100% sure what the line in `conanfile.py` tries to achieve. Does it only patch `find_package(OpenSSL)` to be `REQUIRED`? 

With 0.6.0 on Linux, there should be no direct dependency on `OpenSSL` anymore because it uses whatever your `libcurl` uses for TLS. `OpenSSL` is still a required dependency on Android, though.

**Reply by supervacuus** [reply]
Of course, we are not using the crashpad backend on Android in any capacity.

**Reply by supervacuus** [reply]
In any case, thanks for the reminder. There is no need for `find_package(OpenSSL)` on Linux anymore. I removed it here: https://github.com/getsentry/crashpad/pull/81, and this change will be part of the next release.

**Reply by supervacuus** [reply]
This should be fixed with [0.6.1](https://github.com/getsentry/sentry-native/releases/tag/0.6.1).
